.flowconfig

```diff
@@ -8,6 +8,9 @@
 # Yay for dependencies that are bloated with tests, and .json files that aren't JSON
 .*/node_modules/cjson/test.*
 
+.*/node_modules/config-chain/test/broken.json
+.*/node_modules/npmconf/test/fixtures/package.json
```

Comment:
@sophistifunk is it ok for me to do this? Build fails without it. The changes I made drag in some new deps and flow doesn't seem to like them. One of the new deps has a file named "broken.json" that has badly formatted json and causes flow to barf. Was probably from a test and got published.